packages/core/src/_common/utility.js

```diff
@@ -95,6 +95,23 @@ const isSyntheticsUnavailable = country => countries_without_synthetics.includes
 const blocked_options_countries = ['au'];
 const isOptionsBlocked = country => blocked_options_countries.includes(country);
 
+// Fuction to direct user based on the selected platform from deriv-com before login
+const directUser = platform => {
```

Comment:
Please check if `getPlatformRedirect` in `packages/shared/src/utils/platform/platform.js` is the same as this and use that.